std/range/interfaces.d

```diff
@@ -8,45 +8,45 @@ provides a number of object and $(D interface) definitions that can be used to
 wrap around _range objects created by the $(D std.range) templates.
 
 $(BOOKTABLE ,
-    $(TR $(TD $(D $(LREF InputRange)))
+    $(TR $(TD `$(LREF InputRange)`)
```

Comment:
Now *this* is a clear example of what I was talking about above: why wrap LREF in any kind of code macro, D or backtick, when LREF is already specifically referring to code? It should just output the appropriate thing to begin with and not have any wrapper.